.github/workflows/config.yml

```diff
@@ -0,0 +1,160 @@
+name: SingleStore .NET Connector
+
+on: [push]
+
+env:
+  DOTNET_VERSION: 8.0.405
+  CONNECTOR_VERSION: 1.2.0
+  LICENSE_KEY: ${{ secrets.LICENSE_KEY }}
+  SQL_USER_PASSWORD: ${{ secrets.SQL_USER_PASSWORD }}
+  S2MS_API_KEY: ${{ secrets.S2MS_API_KEY }}
+
+jobs:
+  test-ubuntu:
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        singlestore_image:
+          - singlestore/cluster-in-a-box:alma-8.7.12-483e5f8acb-4.1.0-1.17.15
+          - singlestore/cluster-in-a-box:alma-8.5.22-fe61f40cd1-4.1.0-1.17.11
+          - singlestore/cluster-in-a-box:alma-8.1.32-e3d3cde6da-4.0.16-1.17.6
+          - singlestore/cluster-in-a-box:alma-8.0.19-f48780d261-4.0.11-1.16.0
+          - singlestore/cluster-in-a-box:alma-7.8.9-e94a66258d-4.0.7-1.13.9
```

Comment:
remove versions that are EOL. Using singlestore-labs/singlestore-supported-versions would be even better, but this would come with migration to the dev imageI guess